hack/testing/init-log-stack

```diff
@@ -50,7 +50,7 @@ sudo cp -r ${SERVER_CONFIG_DIR}/master/ /etc/origin/
 
 sudo ansible-playbook -i ansible.inventory \
   -$OS_ANSIBLE_VERBOSITY \
-  $OS_ANSIBLE_DIR/playbooks/common/openshift-cluster/openshift_logging.yml \
+  $OS_ANSIBLE_DIR/playbooks/byo/openshift-cluster/openshift_logging.yml \
```

Comment:
looks like its `playbooks/byo/openshift-cluster/openshift-logging.yml` not `openshift_logging`